### DISCUSSION TITLE
How to log a multiple POTA reference QSO?

### DISCUSSION THREAD

**Discussion by nolith** [magicbug/Cloudlog#3096] [Q&A]
POTA has the 2-fer, 3-fer location where a QSO is valid on multiple references. This is also [supported by the ADIF spec](https://adif.org/314/ADIF_314.htm#POTARefList) but I was not able to mark a single QSO with more than one POTA reference. Am I doing something wrong or this is a missing feature?

**Comment by magicbug** [comment]
Thanks this is an issue and will be resolved
